internal/abci/utils_test.go

```diff
@@ -163,6 +163,11 @@ func (n *fakeNode) GetChainStateByUrl(url string) *api.APIDataResponse {
 	return r
 }
 
+func (n *fakeNode) GetChainDataByUrl(url string) *api.APIDataResponse {
+	//todo implement query data functionality
+	return nil
+}
```

Comment:
I would prefer a panic or `n.t.Fatal(...)`, to make it immediately obvious why the test failed